src/porepy/utils/tangential_normal_projection.py

```diff
@@ -365,3 +365,75 @@ def set_local_coordinate_projections(
         d_l = mdg.subdomain_data(sd_secondary)
         # Store the projection operator in the lower-dimensional data.
         d_l["tangential_normal_projection"] = projection
+
+
+def sides_of_fracture(
+    intf: pp.MortarGrid, sd_primary: pp.Grid, direction: np.ndarray
+) -> tuple[np.ndarray, np.ndarray, bool]:
+    """Identify the top and bottom sides of the interface based on a direction vector.
```

Comment:
You mentioned you wanted to switch top/bottom to positive/negative. Have you decided against this?